editor/filesystem_dock.cpp

```diff
@@ -2346,7 +2346,7 @@ void FileSystemDock::_file_option(int p_option, const Vector<String> &p_selected
 
 		case FILE_COPY_PATH: {
 			if (!p_selected.is_empty()) {
-				String fpath = p_selected[0];
+				const String &fpath = p_selected[0];
 				DisplayServer::get_singleton()->clipboard_set(fpath);
```

Comment:
```suggestion
				DisplayServer::get_singleton()->clipboard_set(p_selected[0]);
```

Reply:
I'm biased against this change -- `fpath = <...>` is a pattern used across `filesystem_dock.cpp` -- I think it adds code patterning and readability to the maintainer.